deltachat-ios/Helper/Protocols.swift

```diff
@@ -1,7 +1,7 @@
 import UIKit
 
 protocol QrCodeReaderDelegate: AnyObject {
-    func handleQrCode(_ code: String)
+    func handleQrCode(_ viewController: UIViewController, qrCode: String)
```

Comment:
the added viewController seems not be be used anywhere at a first glance

Reply:
Good catch, thank you!